src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBrokerHelper.cs

```diff
@@ -153,13 +160,21 @@ public async Task<string> GetBrokerAuthTokenSilentlyAsync(IDictionary<string, st
                 }
             }
 
-            _logger.Info("Android Broker AddAccount didn't return any results. ");
+            _logger.Info("Android Broker didn't return any results.");
```

Comment:
do we want to be more precise (to the silent authentication request) ?

Reply:
There are log messages beforehand that indicate that the silent flow has started but it would not hurt to be more precise here.